functions/Restore-DbaBackup.ps1

```diff
@@ -0,0 +1,215 @@
+function Restore-DbaBackup
```

Comment:
Agreed, it will certainly be a showcase! That's why I asked you to perform the review. I was too excited, which means I wouldn't look as closely, though I will be testing it thoroughly.